modules/ws/src/test/resources/TestImportedTypesWithLongURIUsingRelativePaths.wsdl

```diff
@@ -0,0 +1,76 @@
ted_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../imported_schemas/../TestTypeDefinitionsWithMultipleLevels.wsdl"/>
```

Comment:
What is the location length that causes problems?
Add some comment here explaining this.